### PR TITLE
ci: pin @wdio/browser-runner v8.36.0

### PR DESCRIPTION
**Pull request by aladdin-add**
the browser testing was failing: https://github.com/eslint/eslint/actions/runs/9324019940/job/25668389482. the commit aims to fix it for now, to unblock merging PRs.

#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ x] Other, please explain:

#### Is there anything you'd like reviewers to focus on?

not really.

### CODE REVIEW

File: wdio.conf.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ exports.config = {
                 alias: {
                     util: "rollup-plugin-node-polyfills/polyfills/util",
                     path: "rollup-plugin-node-polyfills/polyfills/path",
+                    "node:path": "rollup-plugin-node-polyfills/polyfills/path",
```
Comment (aladdin-add) [inline wdio.conf.js:21]: to fix the error: https://github.com/eslint/eslint/actions/runs/9343712309/job/25713716963#step:5:47

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Comment by mdjermanovic** [comment]
Looks like now another error appears on `main`:

```js
 "concise" Reporter:
========= Your concise report ==========
chrome-headless-shell (v125.0.6422.112) on linux
❌ Failed to setup tests, no tests found
```

**Comment by aladdin-add** [comment]
re-run the failed job, it's working now😅: https://github.com/eslint/eslint/actions/runs/9347575671/job/25725796524

Let's watch it for a while and see if it shows up again.

**Pull request merged**